kubectl-mcp-app/CONTRIBUTING.md

```
@@ -0,0 +1,159 @@
+```bash
+npm install
+```
+
+3. Build the project:
+```bash
+npm run build
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)